core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2427,7 +2427,6 @@ class ReplicaManagerTest {
 
     // verify that broker 1 did remove its metrics when no longer being the leader of partition 1
     verify(mockTopicStats1).removeOldLeaderMetrics(topic)
-    verify(mockTopicStats1).removeOldFollowerMetrics(topic)
```

Comment:
Why do we need to remove this?

Reply:
@dajac  Because the original code logic partition only appear from Leader To Follower , only the removeOldLeaderMetrics method will be executed. If this line of removeOldFollowerMetrics  code is not removed, the test will fail.
If we want to keep 
`verify(mockTopicStats1).removeOldFollowerMetrics(topic),` 
I think the good way is to modify the method annotation instead of modifying the test.